### PR TITLE
Fix for running MLflow Projects on Windows

### PR DESCRIPTION
**Pull request by juntai-zheng**
## What changes are proposed in this pull request?

Addresses #1670.

## How is this patch tested?

Travis

### Is this a user-facing change?

- [X] No. You can skip the rest of this section.
- [ ] Yes. Give a description of this change to be included in the release notes for MLflow users.

(Details in 1-2 sentences. You can just refer to another PR with a description if this PR is part of a larger change.)

### What component(s) does this PR affect?

- [ ] UI
- [ ] CLI
- [ ] API
- [ ] REST-API
- [ ] Examples
- [ ] Docs
- [ ] Tracking
- [ ] Projects
- [ ] Artifacts
- [ ] Models
- [ ] Scoring
- [ ] Serving
- [ ] R
- [ ] Java
- [ ] Python

### How should the PR be classified in the release notes? Choose one:

- [ ] `rn/breaking-change` - The PR will be mentioned in the "Breaking Changes" section
- [X] `rn/none` - No description will be included. The PR will be mentioned only by the PR number in the "Small Bugfixes and Documentation Updates" section
- [ ] `rn/feature` - A new user-facing feature worth mentioning in the release notes
- [ ] `rn/bug-fix` - A user-facing bug fix worth mentioning in the release notes
- [ ] `rn/documentation` - A user-facing documentation change worth mentioning in the release notes

### CODE REVIEW

File: mlflow/projects/__init__.py [hunk 1]
```diff
@@ -619,7 +619,7 @@ def _get_conda_command(conda_env_name):
         if os.name != "nt":
             return ["source %s %s 1>&2" % (activate_path, conda_env_name)]
         else:
-            return ["conda activate %s" % (conda_env_name)]
+            return ["conda.bat activate %s" % (conda_env_name)]
```
Comment (mparkhe) [inline mlflow/projects/__init__.py:622]: Is it `conda.exe` or `conda.bat`?

cc: @tomasatdatabricks @dbczumar

**Comment by aarondav** [comment]
What's the status of this PR?

File: mlflow/projects/__init__.py (hunk 1 above)
Comment (juntai-zheng) [reply mlflow/projects/__init__.py:622]: Quick bump on this

Comment (tomasatdatabricks) [reply mlflow/projects/__init__.py:622]: What's the background for this. Have you tested it on windows @juntai-zheng ? Can you add a windows test?

**Comment by apurva-koti** [comment]
triggering build

**Pull request closed without merging**